bucket/godot-beta.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "4.0-beta1",
+    "description": "A feature-packed, cross-platform game engine to create 2D and 3D games from a unified interface.",
+    "homepage": "https://godotengine.org/",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.tuxfamily.org/godotengine/4.0/beta1/Godot_v4.0-beta1_win64.exe.zip"
+        },
+        "32bit": {
+            "url": "https://downloads.tuxfamily.org/godotengine/4.0/beta1/Godot_v4.0-beta1_win32.exe.zip"
```

Comment:
Hashes are missing, you can find them here https://downloads.tuxfamily.org/godotengine/4.0/beta1/SHA512-SUMS.txt